packages/core/core/package.json

```diff
@@ -37,6 +37,7 @@
     "core-js": "3.40.0",
     "http-errors": "2.0.0",
     "http-status-codes": "2.3.0",
+    "minimatch": "^10.0.1",
```

Comment:
I'd suggest pin to have more deterministic versions.

Reply:
👍